mysql-test/t/threads_running.test

```diff
@@ -1,21 +1 @@
---source include/master-slave.inc
-
-SHOW STATUS LIKE 'Threads_running';
-SHOW STATUS LIKE 'Threads_binlog_client';
-
-connection slave;
-STOP SLAVE;
-
-connection master;
-
-FLUSH LOGS;
---disable_warnings
-drop table if exists t1, t2;
-create table t1 (a int);
-insert into t1 values (1);
-drop table t1;
---enable_warnings
-FLUSH LOGS;
-
-SHOW STATUS LIKE 'Threads_running';
-SHOW STATUS LIKE 'Threads_binlog_client';
+--source include/threads_running.inc
```

Comment:
This test should belong to rpl_nogtid suite?

Reply:
Agree with @laurynas-biveinis . Please move to `mysql-test/suite/rpl_nogtid/t/threads_running.test`.